src/lib/jsonParseV2/parseStringVector.cpp

```diff
@@ -69,6 +69,17 @@ bool parseStringVector
 
     std::string value = iter->GetString();
 
+   for (rapidjson::Value::ConstValueIterator iter_new = iter+1; iter_new != jsonVector.End(); ++iter_new)
```

Comment:
I think the addition of this code comment can be useful:

```suggestion
   // We start the loop in next position of the vector (note the iter+1), so check that in the elements "to the right" there aren't duplicates
   for (rapidjson::Value::ConstValueIterator iter_new = iter+1; iter_new != jsonVector.End(); ++iter_new)
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4025/commits/ec093717221e85d86e02710201cb026c47f49dbd